packages/create-app/bin/d2-create-app

```diff
@@ -5,4 +5,4 @@ const pkgJson = require('../package.json')
 const command = require(`..`)
 
 notifyOfUpdates(pkgJson)
-makeEntryPoint(command).parse()
+makeEntryPoint(command)
```

Comment:
Do we want to fix these in its own PR? This PR wont work without this fix though.

Reply:
This is fine, thanks for finding the bug!